cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -750,20 +828,28 @@ static wifi_config_t wifi_config_sta = {
  */
 static wifi_config_t wifi_config_ap = {
     .ap = {
+#ifdef ESP_WIFI_SSID
         .ssid = ESP_WIFI_SSID,
-        .ssid_len = ARRAY_SIZE(ESP_WIFI_SSID),
-        .ssid_hidden = 1,               /* don't make the AP visible */
+        .ssid_len = ARRAY_SIZE(ESP_WIFI_SSID) - 1,
+#endif
 #ifdef ESP_WIFI_PASS
         .password = ESP_WIFI_PASS,
         .authmode = WIFI_AUTH_WPA2_PSK,
 #else
         .authmode = WIFI_AUTH_OPEN,
 #endif
-        .max_connection = 0,            /* don't allow connections */
-        .beacon_interval = 60000,       /* send beacon only every 60 s */
+#ifdef MODULE_ESP_WIFI_AP
+        .ssid_hidden = ESP_WIFI_SSID_HIDDEN, /* don't make the AP visible */
+        .max_connection = ESP_WIFI_MAX_CONN, /* maximum number of connections */
+        .beacon_interval = 100,              /* send beacon every 100 ms */
```

Comment:
Does it need to be that often?

Reply:
[Default] is 100 ms for Espressif. Will add a configuration define for it though.

[Default]: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/network/esp_wifi.html#_CPPv4N16wifi_ap_config_t15beacon_intervalE